social/app/forms/author.py

```diff
@@ -0,0 +1,16 @@
+from django import forms
+
+
+class FindRemoteAuthorForm(forms.Form):
+    URI_REGEX = r'^(.+)//(?P<host>([^/]+))/.*author/(?P<pk>[0-9a-fA-F-]+)/?$'
```

Comment:
The trailing slash should be optional according to the [Profile API call specification](https://github.com/abramhindle/CMPUT404-project-socialdistribution/blob/master/example-article.json#L304).